python/rmm/tests/test_rmm_pytorch.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+import rmm
+
+torch = pytest.importorskip("torch")
+
+
+@pytest.fixture(scope="session")
+def torch_allocator():
+    try:
+        from torch.cuda.memory import change_current_allocator
+    except ImportError:
+        pytest.skip("pytorch pluggable allocator not available")
```

Comment:
There's a pytest utility for this if you want to use it. [`pytest.importorskip`](https://docs.pytest.org/en/latest/how-to/skipping.html#skipping-on-a-missing-import-dependency)

Reply:
Does that handle `import`ing specific functions?

This is using `importorskip` for `torch` generally above. The `torch.cuda.memory` module [has been around for a while]( https://github.com/pytorch/pytorch/commits/master/torch/cuda/memory.py ). Though the functionality we need from it is pretty new.

Maybe in the future this could require a specific PyTorch version. There doesn't seem to be one yet that has what we need though.

Reply:
No, it only handles _modules_ and you have to use attribute accessors to get functions. It's kind of messy. The current solution is probably easier to read, let's keep it as-is.